packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -38,11 +55,13 @@
 		"test:end-to-end": "npm run test:jest",
 		"test:jest": "jest",
 		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
-		"test:mocha": "npm run mocha",
+		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.test.*js\" --exit",
+		"test:mocha:esm": "echo \"Skipping ESM tests -- See AB#7470\"",
```

Comment:
Unit tests fails to run under ESM due to the use of require(..) in the 'proxyquire' dependency, so we only test ESM for now.

Reply:
We may need to investigate alternatives to proxyrequire for these test patterns then. Ideally this package could strictly be ESM. I'll file something.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/7581